lib/index.js

```diff
@@ -461,9 +462,9 @@ if (enabled) {
       ao.metrics.start();
     }
 
-    ao.requestStore.run(function () {
+    ao.tContext.run(function () {
       const v = process.versions;
-      const data = {
+      const kvpairs = {
         '__Init': 1,
         'Layer': 'nodejs',
```

Comment:
i don't think they are needed here. removed.

Reply:
btw, this is the kind of thing that requires manual testing now because oboe doesn't send status channel data to the UDP reporter. when i integrate java-collector end-to-end i'll be able to verify this as well.